app/javascript/App.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useEffect } from 'react';
+import { Routes, Route } from 'react-router-dom';
+import { useNavigate } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import NavBar from './pages/NavBar';
+import Home from './components/Home';
+import Landing from './components/Landing';
+import Detail from './components/Detail';
+import Reservations from './components/Reservations';
+import ReservationForm from './components/ReservationForm';
+import Login from './components/Login';
+import ProductForm from './components/ProductForm';
+import { fetchHomes } from './redux/home/Home';
+import Signup from './components/Signup';
+import Products from './components/Products';
+import { fetchUsers } from './redux/registration/Registration';
+
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://user-images.githubusercontent.com/57421823/225416816-0b90b681-1a2d-49f2-8067-71ba46b00de1.png)